mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -283,6 +283,7 @@ def _compute_moc_climo_postprocess(config, runStreams, variableMap, calendar,
                        dictClimo['endYearClimo'])
     if not os.path.exists(outputFileClimo):
         print '   Load data...'
+        chunking = config.getExpression(sectionName, 'maxChunkSize')
```

Comment:
Thanks @milenaveneziani, this was a bug due to a commit that was dropped by accident.  This would have not originally worked for config options that didn't have `maxChunkSize` in the MOC and I've rectified this with a new commit that makes the config parser return `None` for empty options.  I also added a unit test to show this behavior reading from the config file that doesn't have the correct option in a section, i.e., returning `None`.

Now, chunking is `None` so nothing happens and the default behavior is expected, i.e., chunking is not used.